src/ui/server/controllers/insurance/business/contact/index.ts

```diff
@@ -22,6 +23,8 @@ const { CONTACT: CONTACT_TEMPLATE } = TEMPLATES.INSURANCE.EXPORTER_BUSINESS;
 
 export const TEMPLATE = CONTACT_TEMPLATE;
 
+export const CONTACT_FIELDS_IDS = [FIRST_NAME, LAST_NAME, EMAIL, POSITION];
```

Comment:
typo, `FIELD_IDS` - love this though, very nice. Can we add a unit test for this as well?

Reply:
FIELD_IDS is already imported at the top

Reply:
I think we should rename the top import to be `BUSINESS_FIELD_IDS` and then we can have a generic `FIELD_IDS` in all controllers.

Otherwise all controllers will have different naming conventions for their field IDs,